uwsdora.module

```diff
@@ -47,6 +47,18 @@ function uwsdora_ir_citationcmodel_islandora_view_object_alter(AbstractObject &$
     )
   );
   $output['citation.tab']['google_scholar_search']['#markup'] = $link;
+
+  // Adding PDF Download Link
```

Comment:
may want to run drush-no-test-empty uwsdora to catch missed coding standards

Reply:
Ran and fixed all coding standards issues.